packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -104,27 +119,29 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
                 minIncludeTextWidth: 800,
               ),
             ),
+            summaryTreeController?.isExperimentalStoryOfLayoutEnabled == true
```

Comment:
replace `== true` with `?? true`

Reply:
also use the string comprehension `if` so write this as
```dart
if (summaryTreeController?.isExperimentalStoryOfLayoutEnabled)
OutlineButton(
              onPressed: summaryTreeController?.toggleDebugLayoutSummaryEnabled,
              child: Label(
                FlutterIcons.lightbulb,
                'Layout',
                minIncludeTextWidth: 900,
              ))
```
That way you don't need the placeholder SizedBox

Reply:
That is very neat! Just knew about the language feature, thank you Jacob :)